.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: Docs - Build and Deploy
+
+on: [push]
```

Comment:
Please add `pull request` https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

Reply:
Done.